### ISSUE TITLE
Membranous lupus nephritis

### ISSUE THREAD

**Issue by RichardKlaus** [obophenotype/human-phenotype-ontology#6836]
**Preferred term label:**
Membranous lupus nephritis

**Synonyms**
Class V lupus nephritis

**Definition (free text, please give PubMed ID)**
A lupus nephritis subclass with global or segmental subepithelial immune deposits or their morphological sequelae by light microscopy and by immunofluorescence or electron microscopy, with or without mesangial alterations. May occur in combination with class III or IV, in which case both classes are diagnosed. May show advanced sclerosis
PMID: 31974366

**Parent term (use hpo.jax.org/app)**
Lupus nephritis (newly proposed term)

**Related terms**
Mesangial hypercellularity HP:0012574
Mesangial Immune complex deposition HP:0033295
Glomerular sclerosis HP:0000096
Glomerular subepithelial immune-complex deposits HP:0033601
Systemic lupus erythematosus HP:0002725

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
OMIM #152700, ORPHA:536

**Your nano-attribution (ORCID)**
0000-0002-2234-4248

**Comment by pnrobinson** [comment]
added
